CMR/python/demos/notebooks/collections.ipynb

```diff
@@ -27,9 +29,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
outputs need to be cleared before committing

Reply:
Not sure how that got into the PR. Retracting.